ngraph/test/runtime/interpreter/unit_test.manifest

```diff
@@ -10,24 +10,12 @@ INTERPRETER.onnx_top_k_opset_10
 reduce_sum_large_1d_to_scalar
 reduce_sum_keep_large_1d_to_scalar
 
-# "Unsupported op 'Interpolate'
-INTERPRETER.onnx_resize10_down_scales_const_linear
-INTERPRETER.onnx_resize10_down_scales_const_nearest
-INTERPRETER.onnx_resize10_up_scales_const_linear
-INTERPRETER.onnx_resize10_up_scales_const_nearest
-INTERPRETER.onnx_upsample8_nearest_infer
-INTERPRETER.onnx_upsample8_linear_infer
-INTERPRETER.onnx_upsample9_scales_const_nearest_infer
-INTERPRETER.onnx_upsample9_scales_const_linear_infer
-INTERPRETER.onnx_upsample9_scales_input_nearest_infer
-INTERPRETER.onnx_resize11_scales_down_linear
-INTERPRETER.onnx_resize11_scales_up_linear_asymmetric
-INTERPRETER.onnx_resize11_scales_nearest_asymmetric_floor
-INTERPRETER.onnx_resize11_sizes_nearest_asymmetric_floor
-INTERPRETER.onnx_resize11_sizes_linear
-INTERPRETER.onnx_resize11_scales_nearest_asymmetric_floor_dynamic_sizes
+# ONNX Loop
```

Comment:
Remove this and next line since they already present below.

Reply:
Done.